src/assets/base.css

```diff
@@ -1,4 +1,19 @@
-/* color palette from <https://github.com/vuejs/theme> */
```

Comment:
Can you double check how this file is create? if it is copy from vuejs theme, we need to keep the License header without changing it.